examples/mugen/test/retrieval/test_video_clip.py

```diff
@@ -128,7 +129,31 @@ def make_input_video(c_dim=1):
 
         return make_input_video
 
-    def test_forward_trainable(self, utils):
+    def test_forward_pretrained_trainable(self, utils, mocker):
+        make_input_video = utils
+        input_video = make_input_video()
+        patch_load_model(mocker)
+        encoder = VideoEncoder(pretrain_path=get_asset_path("S3D_sample.pt"))
+
+        out = encoder(input_video)
+        expected_sum = 1829.8858
+        assert_expected(
+            actual=out.shape, expected=torch.Size([2, 1024])
+        )  # batch x embedding
+        assert_expected(
+            actual=out.sum(), expected=torch.as_tensor(expected_sum), rtol=0, atol=1e-3
+        )
+        assert next(encoder.parameters()).requires_grad
+        assert encoder.out_dim == 1024
```

Comment:
why need to enforce this value here? are there any subsequent routines that depend on out_dim=1024?

Reply:
wanted to check that the out_dim logic was correct